app/src/components/transaction.svelte

```diff
@@ -21,7 +22,8 @@
       const pays = await CLN.list_pays(tag);
       const trans = parseClnPayments(pays.payments);
       trans.reverse();
-      transactions = [...trans];
+      //transactions = [...trans];
+      transactions = { index: 0, invoice: "abcdefg", date: "10,24,2025", amount: 55555}
```

Comment:
change this hardcoded value back

Reply:
resolved in [c9381df](https://github.com/stakwork/sphinx-swarm/pull/470/commits/c9381df3b96e34cc560a5849063a5661390ef81e)